socorro/signature/tests/siglists/test-invalid-sig-list.txt

```diff
@@ -1,4 +1,4 @@
 # Test Signatures List.
 # Contains an invalid regular expression.
 
-lol[[wut\]
+lol[wut\]
```

Comment:
This kicked up a warning in Python 3 when compiled as a regexp. Since it's supposed to be invalid, but not kick up a warning (which fails our pytest setup), I adjusted it.